RallyHereIntegration/Source/RallyHereIntegration/Private/RH_GameInstanceBootstrappers.cpp

```diff
@@ -784,7 +810,7 @@ void URH_GameInstanceServerBootstrapper::OnReservationComplete(bool bSuccess)
 	bool bStartedHelper = false;
 	FString SessionType = DefaultAutoCreateSessionType;
 
-	if (FParse::Value(FCommandLine::Get(), TEXT("rhsessiontype="), SessionType))
+	if (FParse::Value(FCommandLine::Get(), TEXT("rh.sessiontype="), SessionType))
```

Comment:
Is this an intentional breaking change?  Should we support both?

Reply:
It was to make things more consistent with our other non-GHA parameters, but I am not sold on it tbh.  Will remove it for now as this PR has a bunch of more important things.